src/routes/CreateNewSafePage/CreateNewSafePage.test.tsx

```diff
@@ -0,0 +1,862 @@
+    })
+  })
+  describe('Step 2: Safe Name', () => {
+    it('Shows Safe Name text Input and the current network', async () => {
+      const customState = {
+        providers: {
+          name: 'MetaMask',
+          loaded: true,
+          available: true,
+          account: '0x680cde08860141F9D223cE4E620B10Cd6741037E',
+          network: '4',
+          smartContractWallet: false,
+          hardwareWallet: false,
+        },
+      }
+
+      render(<CreateNewSafePage />, customState)
+      await waitForElementToBeRemoved(() => screen.getByTestId('create-new-safe-loader'))
+
+      fireEvent.click(screen.getByText('Continue'))
+      await waitFor(() => expect(screen.getByTestId('create-safe-name-step')).toBeInTheDocument())
+
+      expect(screen.getByTestId('create-new-safe-name-field')).toBeInTheDocument()
+      expect(screen.getByText('Rinkeby')).toBeInTheDocument()
+    })
+
+    it('goes to the next Owners and Confirmations step when clicks on the next button', async () => {
+      const customState = {
+        providers: {
+          name: 'MetaMask',
+          loaded: true,
+          available: true,
+          account: '0x680cde08860141F9D223cE4E620B10Cd6741037E',
+          network: '4',
+          smartContractWallet: false,
+          hardwareWallet: false,
+        },
+      }
+
+      render(<CreateNewSafePage />, customState)
+      await waitForElementToBeRemoved(() => screen.getByTestId('create-new-safe-loader'))
+
+      fireEvent.click(screen.getByText('Continue'))
+      await waitFor(() => expect(screen.getByTestId('create-safe-name-step')).toBeInTheDocument())
+
+      fireEvent.click(screen.getByText('Continue'))
+      await waitFor(() => expect(screen.getByTestId('create-safe-owners-confirmation-step')).toBeInTheDocument())
+    })
+  })
+
+  describe('Step 3: Owners and Confirmations', () => {
+    it('Shows user Account as a default owner', async () => {
+      const customState = {
+        providers: {
+          name: 'MetaMask',
+          loaded: true,
+          available: true,
+          account: '0x680cde08860141F9D223cE4E620B10Cd6741037E',
+          network: '4',
+          smartContractWallet: false,
+          hardwareWallet: false,
+        },
+      }
+
+      render(<CreateNewSafePage />, customState)
+      await waitForElementToBeRemoved(() => screen.getByTestId('create-new-safe-loader'))
+
+      fireEvent.click(screen.getByText('Continue'))
+      await waitFor(() => expect(screen.getByTestId('create-safe-name-step')).toBeInTheDocument())
+
+      fireEvent.click(screen.getByText('Continue'))
+      await waitFor(() => expect(screen.getByTestId('create-safe-owners-confirmation-step')).toBeInTheDocument())
+
+      const userAccount = customState.providers.account
+      const defaultOwner = screen.getByTestId('owner-address-0') as HTMLInputElement
+      expect(defaultOwner.value).toBe(userAccount)
+    })
+
+    it('At least one owner is Required', async () => {
```

Comment:
Just a minor thing but the idea of this `it` function is that it's supposed to read like a full sentence.
E.g. `it("makes a cup of coffee", () => ☕️)`